package.json

```diff
@@ -54,7 +54,7 @@
     "fxa-checkbox": "git://github.com/mozilla/fxa-checkbox.git#7f856afffd394a144f718e28e6fb79092d6ccddd",
     "fxa-crypto-relier": "2.1.0",
     "fxa-geodb": "1.0.0",
-    "fxa-js-client": "1.0.1",
+    "fxa-js-client": "git://github.com/mozilla/fxa-js-client.git#session-reauth",
```

Comment:
Tagged a new version to use, https://github.com/mozilla/fxa-js-client/releases/tag/1.0.2.